### PR TITLE
Weekly 15 reflection and d5

### LINKED ISSUE #27: Weekly Reflection 15
This is for Weekly Reflection 15 and the rest of the changes to D5

### PR DESCRIPTION
**Pull request by HadesGeneral**

### CODE REVIEW

File: Reflections/Week_15_Reflection.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  - Finding a second device to record things since the app is on my phone. One of the professors gave me a camera and a tablet to allow me to do this now. The main difficulty now is finding time to do the rest of what is needed.
+3. What do you plan to accomplish in the following week? (Please list at least 3 concrete "S.M.A.R.T." goals, along with estimated number of hours to complete, or day to complete)
+  - film and do the rest of the End-to-End Testng (continuous, 2+ hours)
+    - Specific: I need to test the entire system with the goal of it to be focused on the turnability of the entire system.
+    - Measurable: i will be using a measuring tape and cones to help figure out how much it can turn. 
+    - Achievable: Sort of, the main conflict is finding time and the space. The space I have been using is the Berea park and it's parking lot. It isn't alwasy free to use.
+  - Test wheels with gravel. (30 minutes)
+    - Specific: I got to record the system trying to run over gravel. To see if it is possible, if so, how easily does it go over it. 
+    - Measurable: Yes, if it struggles at all then it is not an ideal surface for the system to run on.
```
Comment (belainehi) [inline Reflections/Week_15_Reflection.md:13]: How do you define struggling? Do you have a metric for what's struggling vs what's not for when you test your system on gravel?

File: Reflections/Week_15_Reflection.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+1. What have you accomplished this week? (Please list each accomplishment, and explain) 
+  - I was able to create and add in a lot of details for the Unit Testing (shown in D5)
+  - Changed Cognative Walkthrough to vertical End-to-End testing. Did the outline and did one type of testing. (shown in D5)
+2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+  - Finding a second device to record things since the app is on my phone. One of the professors gave me a camera and a tablet to allow me to do this now. The main difficulty now is finding time to do the rest of what is needed.
+3. What do you plan to accomplish in the following week? (Please list at least 3 concrete "S.M.A.R.T." goals, along with estimated number of hours to complete, or day to complete)
+  - film and do the rest of the End-to-End Testng (continuous, 2+ hours)
+    - Specific: I need to test the entire system with the goal of it to be focused on the turnability of the entire system.
+    - Measurable: i will be using a measuring tape and cones to help figure out how much it can turn.
```
Comment (belainehi) [inline Reflections/Week_15_Reflection.md:9]: I see what you are saying here but the question is asking how you will know if you achieved the goal of testing your system, what's the standard that tells you if you have successfully performed the test? Here you mentioned using tape and cones but how will they be used to measure how much your skateboard can turn?

File: Reflections/Week_15_Reflection.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    - Specific: I need to test the entire system with the goal of it to be focused on the turnability of the entire system.
+    - Measurable: i will be using a measuring tape and cones to help figure out how much it can turn. 
+    - Achievable: Sort of, the main conflict is finding time and the space. The space I have been using is the Berea park and it's parking lot. It isn't alwasy free to use.
+  - Test wheels with gravel. (30 minutes)
+    - Specific: I got to record the system trying to run over gravel. To see if it is possible, if so, how easily does it go over it. 
+    - Measurable: Yes, if it struggles at all then it is not an ideal surface for the system to run on. 
+    - Achievable: This is achievable, just need to find a place and record.
+  - Create and finish Presention slides(5+ hours)
+    - Specific: I need to create and finish the presentation on google slides for the final of the class. 
+    - Measurable: Once I have everything needed in the Rubric for the class, then i know i have completed it, or near finish.
```
Comment (belainehi) [inline Reflections/Week_15_Reflection.md:17]: What is needed for your particular presentation to be completed?

File: Reflections/Week_15_Reflection.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    - Achievable: This is since I have already started on creating it and it is needed to finish the course.
+4. What resources will you use to accomplish your goals for the upcoming week? (please list out the resources)
```
Comment (belainehi) [inline Reflections/Week_15_Reflection.md:19]: Great job listing out all your components!

**Review by belainehi — CHANGES_REQUESTED** [review]
I have finished reviewing this reflection. Well done on listing out all of your components and tools. One thing that can be improved is adding more details to the measurable goals. In your testing and goals there needs to be a clearly defined objective metric to tell you what's successful and what is not and to what degree something is successful. It is not enough to say that the system fails the gravel test if it struggles, what does it mean if it struggles? How do you measure it?

**Comment by HadesGeneral** [comment]
Truthfully, I don't know how to describe when it fails other than it looses grip or won't move anymore. Idk how else to measure the system. I've done all the testing I can, I'm focused on the presentation now.

-- Brian Cox

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Pull request merged**